### DISCUSSION TITLE
Internal Server Error (500) When Submitting Availability Data

### DISCUSSION THREAD

**Discussion by migueljcpinto** [neuefische/questions#712] [Web] [answered]
Hello!

I'm facing an issue with my project. This code used to work correctly, but after I changed a few things and went back, this error started to appear. When I try to submit availability data, I receive a 500 (Internal Server Error). The error occurs when making a POST request to /api/availability.

The data being sent for availability includes nurseId, type, date, and shift. I've provided two route handlers for availability. The first handles GET and DELETE methods, while the second handles GET and POST methods.

Console logs show the received data correctly, but the error persists.

```
Received availability data: {
  nurseId: '64ec732006b8dda857bb6baa',
  type: 'availability',
  date: '2023-09-20T09:38:47.000Z',
  shift: 'Morning'
}
```

[PR](https://github.com/MiketheBite/Capstone-project/pull/17)

I thank you in advance for insights or suggestions on what might be causing this error and how to resolve it.

**Comment by migueljcpinto** [comment, marked as answer]
I've found out why.

**Reply by Kheirah** [reply]
Tell us more! 😆

**Reply by migueljcpinto** [reply]
I didn't like the fact that the shift was shown in small letters on the display, so I lazily tried to change the "value" to large print, among other things, and that gave me the error, of course!

` <option value="" disabled>
              Select a shift
            </option>
            <option value="morning">Morning</option>
            <option value="afternoon">Afternoon</option>
            <option value="night">Night</option>`
